src/api/app/views/webui2/webui/package/_bottom_actions.html.haml

```diff
@@ -1,58 +1,32 @@
 -# FIXME: Refactor this file
 %ul.list-inline.mb-0
   - if bugowners_mail.present? && configuration['bugzilla_url']
-    %li.list-inline-item
-      = link_to(bugzilla_url(bugowners_mail, "#{project.name}/#{package.name}: Bug"), class: 'nav-link') do
-        %i.fas.fa-bug.text-danger
-        Report Bug
+    = render partial: 'webui2/webui/package/bottom_actions/bugzilla_owner', locals: { bugowners_mail: bugowners_mail,
+                                                                package_name: package.name, project_name: project.name }
   - unless User.current.is_nobody?
     - if current_rev
-      %li.list-inline-item
-        = link_to('#', class: 'nav-link', data: { toggle: 'modal', target: '#branch-modal' }) do
-          %i.fas.fa-code-branch.text-primary
-          Branch package
-      %li.list-inline-item
-        = link_to(package_submit_request_dialog_path(project: project, package: package, revision: revision), remote: true, class: 'nav-link') do
-          %i.fas.fa-share-square.text-obs-yellow
-          Submit package
+      = render partial: 'webui2/webui/package/bottom_actions/branch_package'
+      = render partial: 'webui2/webui/package/bottom_actions/submit_package', locals: { package: package, project: project,
+                                                                   revision: revision }
     - if User.current.can_modify?(package)
-      %li.list-inline-item
-        = link_to(package_edit_path(project: project, package: package, spec_count: spec_count), class: 'nav-link') do
-          %i.fas.fa-edit.text-info
-          Edit description
-      %li.list-inline-item
-        = link_to('#', class: 'nav-link', data: { toggle: 'modal', target: '#delete-modal' }) do
-          %i.fas.fa-times-circle.text-danger
-          Delete package
+      = render partial: 'webui2/webui/package/bottom_actions/edit_description', locals: { project: project, package: package, spec_count: spec_count }
+      = render partial: 'webui2/webui/package/bottom_actions/delete_package'
+
       - Feature.with(:kiwi_image_editor) do
         - if package.kiwi_image? && policy(package).update?
-          %li.list-inline-item
-            = link_to(import_kiwi_image_path(package.id), id: 'edit-kiwi', class: 'nav-link') do
-              %i.fas.fa-compact-disc.text-primary
-              View Image
+          = render partial: 'webui2/webui/package/bottom_actions/view_kiwi', locals: { package_id: package.id }
+
       - if Feature.active?(:cloud_upload) && !User.current.is_nobody?
```

Comment:
`&& !User.current.is_nobody?` is not needed here because it's already checked above in `unless User.current.is_nobody?`